janno_details.md

```diff
@@ -45,63 +46,63 @@ The `Relation_Note` column allows to add free-text information about the relatio
-- If the sample is a modern reference sample (`Date_Type = modern`): `Date_BC_AD_Median`, `Date_BC_AD_Start`, `Date_BC_AD_Stop` should all be set to the value 2000, for 2000AD.
+- If only contextual (e.g. from archaeological typology) age information is available (`Date_Type = contextual`): `Date_BC_AD_Start` and `Date_BC_AD_Stop` should simply report the approximate start and end date determined by the respective source of scientific authority (e.g. an archaeologist knowledgable about the relevant typological sequences). In this case `Date_BC_AD_Median` should be calculated as the mean of `Date_BC_AD_Start` and `Date_BC_AD_Stop` rounded to an integer value.
+- If the sample is a modern reference sample (`Date_Type = modern`): `Date_BC_AD_Median`, `Date_BC_AD_Start`, `Date_BC_AD_Stop` should all be set to the value 2000, for 2000 AD.
 
-The column `Date_Note` allows to add arbitrary free-text information about the dating of a sample.
+The column `Date_Note` stores arbitrary free-text information about the dating of a sample.
 
 ### Genetic summary data
 
 #### Individual properties
 
-The `Genetic_Sex` column should encode the biological sex as determined from the DNA read distribution on the X and Y chromosome. It only allows for the entries 
+The `Genetic_Sex` column should encode the biological sex as determined from the DNA read distribution on the X and Y chromosome. It only allows for the entries
 
 - `F`: female
 - `M`: male
-- `U`: unknown 
+- `U`: unknown
 
 This limitation stems from the genotype data formats by Plink and the Eigensoft software package. Edge cases (e.g. XXY, XYY, X0, ...) can not be expressed with this format and should be reported as `U` with an additional comment in the free text `Note` field. Genetic sex determination for ancient DNA can be performed for example with Sex.DetERRmine [@Lamnidis2018](https://doi.org/10.1038/s41467-018-07483-5).
 
 The `MT_Haplogroup` column is meant to store the human mitochondrial DNA haplogroup for the respective individual in a simple string. The entry can be arbitrarily precise. A software tool to determine the MT haplogroup is for example Haplogrep [@Schnoeherr2023](https://doi.org/10.1093/nar/gkad284).
 
-The `Y_Haplogroup` column holds the respective human Y-chromosome DNA haplogroup in a simple string. The notation should follow a syntax with the main branch + the most terminal derived Y-SNP separated with a minus symbol (e.g. R1b-P312).
+The `Y_Haplogroup` column holds the respective human Y-chromosome DNA haplogroup in a simple string. The notation should follow a syntax with the main branch + the most terminal derived Y-SNP separated with a minus symbol (e.g. R1b-P312), following XXXX naming system.
```

Comment:
The question is how the _main branch_ itself is named. I believe that depending on the reference dataset used, the branches can have slightly different names, meaning that the same sequence could be `N3a-SNP2` in one nomenclature while being `N1c1-SNP2` in the other.

Reply:
It's neither N1c1 nor N3a, but rather N1a, but it's more complicated. ISOGG hasn't been updated since 2020 and the alpha-numerical system has proven impractical for a haplotypic system the size of even the tiny Y chromosome.
Yfull resides to just using the main letter N (for rough orientation on the tree) follow by the terminal SNP.
I feel we have no other choice than following a similar approach as the Y-SNPs are the key identifier.

Reply:
Then maybe suggeting using the Y-tree nomenclature is already enough, potentially also providing an example. I don't think it is realistic to recode all existing Y haplogroups into that format, or validate the nomenclature in any way though.